ts/utils/assert.ts

```diff
@@ -0,0 +1,5 @@
+export function assert(condition: unknown, msg?: string): asserts condition {
+  if (condition === false) {
+    throw new Error(msg);
+  }
+}
```

Comment:
This is a very useful utility. It needs just a couple of improvements:
- Let's make sure that we can pass any type for the condition, so a `!condition` check should cover every falsy values  (such as `false`, `0`, `null`, `undefined`, `""`).
- Since `msg` is nullable, we should have a default message for the `Error`

```suggestion
export function assert(condition: unknown, msg?: string = "Assertion failed"): asserts condition {
  if (!condition) {
    throw new Error(msg);
  }
}
```
What do you think?

Reply:
Right, good suggestion!